File: JAVA PROGRAMMING/Test71.java
// 71. PRIME NUMBER
import java.util.Scanner;

public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Num Value: ");
        int num = in.nextInt();

        boolean flag = true;
        for (int i = 2; i <= Math.sqrt(num); i++) {
            if (num % i == 0) {
                flag = false;
            }
        }
        if (!flag) {
            System.out.println(num + " Is Not A Prime Number.");
        } else {
            System.out.println(num + " Is A Prime Number.");
        }
        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test71.ts
// 71. PRIME NUMBER
const prime = (num: number): void => {
    let flag: boolean = true;
    for(let i = 2; i <= Math.floor(Math.sqrt(num)); i++) {
        if(num % i === 0) flag = false;
    }
    if(!flag) console.log(`${num} Is Not A Prime Number.`);
    else console.log(`${num} Is A Prime Number.`);
}
prime(29);
prime(28);
prime(101);
prime(102);